AAI/FAQ.md

```diff
@@ -238,7 +238,157 @@ client <- clearing : Client is given data
 {% endplantuml %}
 
 
-<hr style="border: 2px solid; margin: 2em auto;"/>    
+{% hr2 %}
+
+## Trust
+
+### What's with all the signed passports and visas etc? Why so complex?
+
+The practical operation of a loosely coupled
+federated ecoystem like GA4GH Passports requires establishing trust
+relationships between the participating entities.
+
+Any entity that is asked to make a decision about sharing data needs to have apriori
+made the decision "which other entities do I trust?". When presented with a
+Passport and Visas from entities that they trust - they can rely on the information (claims) provided
+to make data sharing decisions. If presented with information from entities that are
+untrusted - the content of the message is irrelevant as there is no basis on which to believe
+the content.
+
+So we can see that trust is a crucial element of a working federation. How do we establish
+these trust relationships?
+
+GA4GH Passports and Visas leverage the mechanisms present in JWT (ref) as used
+by the OIDC standards (ref) to cryptographically "sign" tokens. Signed tokens can be
+"verified" using public/private keys.
+
+
+### Why are the Visa claims formatted as JWTs inside the Passport?
+
+A section on the importance to the NiH of retaining original authority in visas.
+
+
+### What are the ways that key management is done in practice?
+
+There are a variety of approaches that can be used for key
+management for Passports and Visas. We will first detail those that can be used for Passports
+and then discuss some extra wrinkles for Visas.
+
+For this discussion we assume there is a concrete JWT from issuer `https://issuer.example.org`
+(possibly a Broker *or* a Visa Issuer).
+
+My clearing house service 'trusts' the above issuer to help make data
+access decisions - and that trust is stated probably through some sort of explicit
+configuration. For our example lets imagine it has a hypothetical YAML configuration file
+
+```yaml
+trusted_brokers:
+  - https://issuer.example.org
+  - https://login.broker.com
+
+trusted_visa_issuers:
+  - https://dac.gov.world
+``` 
+ 
+The service now wants to verify a Passport or Visa
+JWT purporting to be from that issuer.
+
+---
+
+#### Use `jku` in the header of the JWT
+
+The JKU is the URL of a JSON file containing the issuers' public keys.
+
+```json
+{
+  "iss": "https://issuer.example.org",
+  "jku": "https://issuer.example.org/public-keys.json",
+  "kid": "key-october-1"
+}
+```
+
+For our concrete example we say that it is a JSON file residing
+at `https://issuer.example.org/public-keys.json`. We should note that
+there is no requirement that the file is hosted at the same
+location as the issuer (it could have been at `https://keys.example.org` for instance).
```

Comment:
Maybe a link to [RFC 7517 "JSON Web Key"](https://datatracker.ietf.org/doc/html/rfc7517) defining the format of the file would be useful.

Reply:
This was just me attempting to clarify that when doing JWKS it is not necessary that everything comes from the same host (that the key sets can be shared across an entire domain for instance). This is just a FAQ entry so it doesn't have to be as precise as the specification - it can have extra 'colour' if that helps with the user understanding things. But happy to remove as well..